post/merge-scalar-data-arrays/mergeDataArraysIntoOne.py

```diff
@@ -0,0 +1,62 @@
+#
+# \copyright
+# Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+#            Distributed under a Modified BSD License.
+#              See accompanying file LICENSE.txt or
+#              http://www.opengeosys.org/project/license
+#
+
+#!/usr/bin/env python2
```

Comment:
Thanks for dropping the script first of all.

I'm only wondering: Shouldn't that be the very first line of the file?